README.md

```diff
@@ -141,6 +141,16 @@ from your bash compatible shell.  This does the following things:
 * Runs all of our [phpunit](https://phpunit.de/) unit tests.
 * Performs static analysis with [Phan](https://github.com/phan/phan), [Psalm](https://psalm.dev/) and [PHPStan](https://phpstan.org/user-guide/getting-started)
 
+### Other PHP versions
+
+We aim to support PHP 7.4, 8.0 and 8.1 - the developer image `ghcr.io/open-telemetry/opentelemetry-php/opentelemetry-php-base` is tagged as `7.4`, `8.0` and `8.1` respectively, with `7.4` being the default.
```

Comment:
"support officially supported PHP versions" per https://www.php.net/supported-versions.php ??

Reply:
Yap, however it would be nice if you replace "per" with "according to". I'm not trying to nit-pick, but despite the loanword "per" being used like that in English, this is not what the Latin word "per" means, so using it in that way often creates confusion for non native English speakers.